docs/provisioners/ansible-local.mdx

```diff
@@ -109,9 +109,478 @@ The reference of available configuration options is listed below.
 
 Note that one of `playbook_file` or `playbook_files` is required.
 
-@include '/provisioner/ansible/Config-not-required.mdx'
+<!-- Code generated from the comments of the Config struct in provisioner/ansible/provisioner.go; DO NOT EDIT MANUALLY -->
```

Comment:
This is super cool to have this because it's easy to know where to edit stuff 🤔 . But we don't know where this ends, maybe we should add an end statement ?

Reply:
I like it as well and the end statement is a great idea!!